app/models/user.rb

```diff
@@ -33,4 +33,10 @@ def is_member_or_admin?
     admin? || is_member?
   end
 
+  def is_company_owner?(this_company_number)
```

Comment:
You don't need this method.

a user can only be associated with a company via a membership_application.
Rails has set the association; it does so by setting the company_id.
(The association is set when a membership_application is accepted.  see the MembershipApplicationController for that.)

In effect, with this method you are checking to see if Rails has worked.

(and what you found in working on this PR is that the imported data is wrong.)